### ISSUE TITLE
[Photon] Don't apply styles to UI elements that weren't affected by Mozilla's Proton redesign

### ISSUE THREAD

**Issue by nbmrjuhneibkr** [black7375/Firefox-UI-Fix#174]
I've noticed that Lepton changes certain UI elements that looked identical in both original Photon and Proton.
A noticeable example (but probably not the only one) would be the "Open/save file" window:
![0](https://user-images.githubusercontent.com/8460563/129667554-626f235c-ff5f-4066-93d1-39753ca3e681.png)
Why is it white now? Why is it bigger? How is this an improvement over... anything?

I consider this customization to be completely unnecessary, at least in Photon style. Most users choose Photon is to keep UI as familiar as possible. The last thing I'd expect from it is to change things that even Mozilla didn't change in the redesign.

**Comment by black7375** [comment]
One of the goals of this project is to keep `Proton` UI consistent.
That's why it's being applied to a lot of pages that haven't applied design `Proton` yet, and so is design download popup.
https://github.com/black7375/Firefox-UI-Fix/blob/ae2d5c89d02e7b6766478b422100547a5599e4e4/userContent.css#L725-L1012

Also, in the long run, I'm planning to change something like the following mockup.
https://github.com/black7375/Firefox-UI-Fix/issues/74#issue-915707320

---

Currently, it can be removed refer:
https://github.com/black7375/Firefox-UI-Fix/blob/ae2d5c89d02e7b6766478b422100547a5599e4e4/userChrome.css#L402-L436

**Comment by nbmrjuhneibkr** [comment]
Original design goal may no longer match the purpose that the project actually serves for a lot of users. And if there was only one design goal, there wouldn't have been multiple variants to begin with.

If something as minor as [this](https://github.com/black7375/Firefox-UI-Fix/issues/154#issuecomment-899972419) is still considered an issue that needs to be addressed (albeit a low-priority one), then I'm going to assume that consistency with original Photon is also a design goal for the `Photon` variant of Lepton.

Most people who can tolerate drastic UI changes are probably using original Lepton or Proton style - not Photon. The mockup shown on Reddit is definitely not something I'd ever want to see in the `Photon` variant.

I think that it would be beneficial to keep a variant that omits all changes that don't match any current or previous design used by Mozilla.
If you're not going maintain such variant - please, make a permanent wiki entry on how to remove all of such additional customizations manually.

**Comment by nbmrjuhneibkr** [comment]
```
Firefox-UI-Fix/userContent.css
Lines 725 to 1012 in ae2d5c8
```
This fixes buttons, but window is still white.
_____
I genuinely believe that changes like this deviate from the _"let's fix/undo mistakes in the existing design"_ goal, and get dangerously close to _"let's change something just for the sake of changing it"_ - which is exactly why so many people were dissatisfied with Mozilla's redesign in the first place.

**Comment by black7375** [comment]
This project is based on proton, to make up for or improve the shortcomings, and a perfect reproduction is not the goal.

> https://github.com/black7375/Firefox-UI-Fix#why-proton
> - Nice color scheme
> - Modal window & Scrollbar!!

Color scheme and modal window belongs to the target of definite change, not to change for change.

That code is providing the dark mode color.

**Comment by nbmrjuhneibkr** [comment]
Deleted a few more lines, now it looks normal.

https://github.com/black7375/Firefox-UI-Fix/blob/ae2d5c89d02e7b6766478b422100547a5599e4e4/userChrome.css#L390-L437

This really needs to be in the wiki, especially if you're going to introduce changes that further deviate from official Photon _and_ Proton.

**Comment by black7375** [comment]
I'll add it when I get home because it's an inaccessible class environment for the next few hours.

If you want to be reflected more quickly
Anyone can modify it because it's Wiki, and you can add it.
